src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2108,6 +2110,10 @@ def aks_agentpool_add(cmd,      # pylint: disable=unused-argument,too-many-local
     if node_vm_size is None:
         node_vm_size = "Standard_D2s_v3"
 
+    upgradeSettings = AgentPoolUpgradeSettings()
```

Comment:
I might bump this to the first line. Looks like future operations will likely follow this pattern of grabbing a "standard" agentpool upgrade settings and future work will likely modify it as well based on settings passed in from the user.

Reply:
moved up thought it was pretty close to the top.